app/views/admin/products/new.html.erb

```diff
@@ -1,3 +1,21 @@
 <h1>New Product</h1>
 
-<%= render partial: 'product_form' %>
```

Comment:
Good job standardizing all the input forms Ash! By the way, are we not using this partial file anymore?